app/src/main/res/layout/item_user.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_user` appears to be unused
 Unused resources make applications larger and slow down builds.